ui/src/message_popup/popup_question.jsx

```diff
@@ -193,17 +195,25 @@ export default React.createClass({
             </div>}
         </VelocityTransitionGroup>
         <Snackbar
-          open={seconds >= 90}
+          open={seconds >= (this.props.limitMs/1000)}
           message="Remember, these are supposed to be quick responses."
-          autoHideDuration={10000000}
+          onRequestClose={function(reason){
+            //This empty function is meant to cancel out the closing of the toast without having to use
+            //an extremely large autoHideDuration
+          }}
```

Comment:
👍  This is a good comment to explain this, thanks for talking it out in person too!  If this was something we did a lot, we could factor this out into our own component that wraps `SnackBar` but no need to here.